core/services/relay/evm/chain_reader_test.go

```diff
@@ -77,10 +80,26 @@ func (it *chainReaderInterfaceTester) setupNoClient(t require.TestingT) {
 					MethodReturningUint64Slice: {
 						ChainSpecificName: "GetSliceValue",
 					},
+					MethodReturningSeenStruct: {
+						ChainSpecificName: returnSeenName,
+						InputModifications: codec.ModifiersConfig{
+							&codec.HardCodeConfig{
+								OnChainValues: map[string]any{
+									"BigField": testStruct.BigField.String(),
+									"Account":  hexutil.Encode(testStruct.Account),
+								},
+							},
+						},
+						OutputModifications: codec.ModifiersConfig{
+							&codec.HardCodeConfig{
```

Comment:
Would be nice to have tests that show usage of all other modifers: Rename, Drop, ExtractElemen. This should probably tie in with a Readme for Chain Reader that product teams can use to implement chain reader. 
You can just add a TODO with a ticket number, feel free to assign it to me

Reply:
There's a test for showing each used in config in [chainlink-common](https://github.com/smartcontractkit/chainlink-common/blob/BCF-2612-ChainReader-Next/pkg/codec/config_test.go).

I don't want to need to double-test using everything in all of the chainlink repos.  If you use the parse config from common, that'll suffice